src/lib/stitching/kaws/stitching.ts

```diff
@@ -7,7 +7,7 @@ export const kawsStitchingEnvironment = (
   // The SDL used to declare how to stitch an object
   extensionSchema: `
     extend type Artist {
-      marketingCollections: [MarketingCollection]
+      marketingCollections(size: Float): [MarketingCollection]
```

Comment:
I know nothing about what ‘size’ means here (and perhaps that means some docs are missing), but shouldn't this be an integer?

Reply:
Commented in https://github.com/artsy/kaws/pull/62#discussion_r256366023 , but I typed this to match what was there.

This is a function of that lib being used in KAWS to generate schema, and I couldn't figure out how to generate anything but a number/float. (As an aside, it's confusing and led to overhead since how we have Ruby apps w/ GraphQL schemas but _also_ a Typescript app that uses libraries that are wholly unique to itself and not used anywhere else at Artsy).